tests/RESTAPI/schemaValidation/swagger.json

```diff
@@ -0,0 +1 @@
":"header","name":"Authorization","description":"The JWT token.","required":true},{"in":"body","schema":{"properties":{"name":{"type":"string","description":"Name of the rule."},"comment":{"type":"string","description":"Comment of the rule."}},"type":"object","required":["name","comment"]}}],"responses":{"200":{"schema":{"properties":{},"type":"object","required":[]}}}}},"/v1/status":{"get":{"tags":["Status"],"summary":"GetStatus","description":"Get the status of the elements of the backend","consumes":["application/json"],"produces":["application/json"],"parameters":[{"in":"body","schema":{"properties":{},"type":"object","required":[]}}],"responses":{"200":{"schema":{"$schema":"http://json-schema.org/draft-04/schema#","title":"Success-Response:","type":"object","properties":{"connections":{"type":"object","properties":{"database":{"type":"boolean","description":"true if the connection to the database is OK"}},"required":["database"]},"crontasks":{"type":"array","description":"The status of the crontasks."}},"required":["connections","crontasks"]},"description":"Success-Response:"}}}},"/v1/userparams":{"get":{"tags":["User"],"summary":"GetUserparams","description":"Get the userparams types","consumes":["application/json"],"produces":["application/json"],"parameters":[{"type":"string","in":"header","name":"Authorization","description":"The JWT token.","required":true},{"in":"body","schema":{"properties":{},"type":"object","required":[]}}],"responses":{"200":{"schema":{"$schema":"http://json-schema.org/draft-04/schema#","title":"Success-Response:","type":"object","properties":{"itemlist":{"type":"object","properties":{"id":{"type":"number","description":"The id of type (see type endpoint) for manage itemlist userparam."},"properties":{"type":"object","properties":{"typeId":{"type":"number","description":"The property id where store for what id of item manage this userparams."},"elementsPerPage":{"type":"number","description":"The property id for store the default number of elements to display per page."},"propertiesOrder":{"type":"number","description":"The property id for store the order to display properties."},"propertiesHidden":{"type":"number","description":"The property id for store the hidden properties."}},"required":["typeId","elementsPerPage","propertiesOrder","propertiesHidden"]}},"required":["id","properties"]},"csvimport":{"type":"object","properties":{"id":{"type":"number","description":"The id of type (see type endpoint)."},"properties":{"type":"object","properties":{"typeId":{"type":"number","description":"The property id where store for what id of item manage this userparams."},"mappingCols":{"type":"number","description":"The property id for store the columns mapping."},"joiningFields":{"type":"number","description":"The property id for store what fields are joining."}},"required":["typeId","mappingCols","joiningFields"]}},"required":["id","properties"]}},"required":["itemlist","csvimport"]},"description":"Success-Response:"}}}}}}
```

Comment:
what is this file?

Reply:
It's in preparation for tests about schema and documentation of the API, but not yet ready, I delete it.